Automattic/wp-calypso
# Site Address Change: Update network nomenclature from "rename"

Note this is targeted on the branch in #23492, not master.

Change the network API slugs to match the new format.

## To Test

* Apply D11414-code & sandbox the API
* Run this branch
* Browse to http://calypso.localhost:3000/domains/manage
* Attempt various site addresses
  * The validation should be identical to the branch in #23492
